docs/book/src/developers/development.md

```diff
@@ -157,7 +157,7 @@ development will span both CAPZ and CAPI, then follow the [CAPI and CAPZ instruc
   <h2>Warning</h2>
```

Comment:
The warning doesn't seem to be displaying like other warnings elsewhere:
- Possibly incorrect display - https://deploy-preview-5470--kubernetes-sigs-cluster-api-provider-azure.netlify.app/developers/development#tilt-for-dev-in-capz
- https://deploy-preview-5470--kubernetes-sigs-cluster-api-provider-azure.netlify.app/getting-started#warning

Reply:
Good catch. I updated the warning to a `<h1>`, so it should be appearing the same now.